src/component/GreetingForm.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+function GreetingForm() {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    alert(`Hello ${firstName} ${lastName}!`);
+  };
```

Comment:
You can put the handle for empty value and trim the text for better user experience.